ownCloudSDK/Resources/ru_RU.lproj/Localizable.strings

```diff
@@ -0,0 +1,485 @@
+/* 
```

Comment:
This one should be removed as well - right?

Reply:
+1

Reply:
Yup.